sdkproject/Assets/Mapbox/Unity/Editor/FeatureBehaviourEditor.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEditor;
+using Mapbox.Unity.MeshGeneration.Components;
+
+[CustomEditor(typeof(FeatureBehaviour))]
```

Comment:
But isn't that also harder to use compared to a huge button in the inspector? we'll eventually add more in that inspector anyway, that's the script keeping feature data on game objects so pretty much a core element.